packages/@tailwindcss-cli/src/commands/build/index.ts

```diff
@@ -49,6 +49,10 @@ export function options() {
       description: 'Minify the output',
```

Comment:
Should we say "Optimize and minify the output"? Now that the `--optimize` option exists, users may be confused that optimization isn't mentioned here.